students/ChrisH/lesson03/list_lab.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -----------------------------------------------------------
+# list_lab.py
+#  demonstrates basic Python lists
+# -----------------------------------------------------------
+
+# Series 1
+print('*** Series 1')
+basket = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(basket)
+response = input("Please type the name of another fruit > ")
+basket.append(response)
+print(basket)
+response = int(input("Please type a number to display the corresponding fruit > "))
+if 0 < response <= len(basket):
+    print(response)
+    print(basket[response - 1])
+else:
+    print("No fruit at that index.")
```

Comment:
Can you think of how you might wrap this instead in a `while` loop so it will continue to ask for an index?

Reply:
Added a While True loop to this section.